commands/check_cloudformation.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+import click
+
+
+from commands.cloudformation_checks.lint import lint
+
+
+@click.group()
+def copilot():
+    pass
+
+
+def valid_checks():
```

Comment:
I wonder if there's a way to do this within click.

So:
`copilot-helper check` runs the full suite of tests
`copilot-helper check lint waf cfn` runs individual tests (that's click command chaining AFAIK)

would be valid, assuming lint, waf and cfn are part of the check command group.  But you can also run:

`copilot-helper check` to execute the full set of check.

I'd certainly favour leveraging click to do the command routing, validation etc. 

https://click.palletsprojects.com/en/8.1.x/commands/#group-invocation-without-command
https://click.palletsprojects.com/en/7.x/commands/#multi-command-pipelines

Reply:
It now allows for running multiple checks at once

Reply:
You're probably still better leveraging click to do this in the example I showed.  It does useful stuff like automatically renders the doc string as help text etc. and  it seems simpler.

Reply:
Here's the example:

```
#!/usr/bin/env python

import click


@click.group(invoke_without_command=True, chain=True)
@click.pass_context
def check(ctx):
    if ctx.invoked_subcommand is None:
        # https://click.palletsprojects.com/en/7.x/advanced/#invoking-other-commands
        ctx.invoke(lint)
        ctx.invoke(cfn)
        ctx.invoke(waf)


@check.command()
def lint():
    click.echo("Lint check")


@check.command()
def cfn():
    click.echo("CFN check")


@check.command()
def waf():
    click.echo("WAF check")


if __name__ == "__main__":
    check()

```

Reply:
Yes, I think that makes sense. It's the next thing to play with.

Reply:
I think they're saying it isn't advised for one sub command to kick off another sub command, which the example shows. This would be harder to understand from a usability perspective.  But I'm surprised there isn't already logic for a group to run all sub commands.